.changelog/26699.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+client: restore task network status on client restart
```

Comment:
It might be worth expanding this, so users have an idea of what bug and impact on them we are solving.